bundles/WebToPrintBundle/src/Installer.php

```diff
@@ -0,0 +1,122 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\WebToPrintBundle;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use Pimcore\Db;
+use Pimcore\Extension\Bundle\Installer\SettingsStoreAwareInstaller;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Installer extends SettingsStoreAwareInstaller
+{
+
+    protected const USER_PERMISSION_CATEGORY = 'Pimcore Web2Print Bundle';
+    protected const USER_PERMISSIONS = [
+        'web2print_settings',
+    ];
+
+    protected const STANDARD_DOCUMENT_ENUM_TYPES = [
+        'page',
+        'link',
+        'snippet',
+        'folder',
+        'hardlink',
+        'email',
+        'newsletter'
+    ];
+
+    protected const BUNDLE_EXTRA_DOCUMENT_ENUM_TYPES = [
+        'printpage',
+        'printcontainer'
+    ];
+
+    public function install(): void
+    {
+        $this->installDatabaseTable();
+        $enums = array_merge($this->getCurrentEnumTypes(), self::BUNDLE_EXTRA_DOCUMENT_ENUM_TYPES);
+        $this->modifyEnumTypes($enums);
+        $this->addUserPermission();
+        parent::install();
+    }
+
+    public function uninstall(): void
+    {
+        // Only remove permissions. Cleanup can be done by dev or command
+        $this->output->writeln("<info>Uninstalling only removes permissions. To clean up all documents and dependencies, please install the bundle again and run bin/console pimcore:documents:cleanup printpage printcontainer</info>");
```

Comment:
```suggestion
        $this->output->writeln("<info>Uninstalling only removes permissions. To clean up all documents and dependencies, please run 'bin/console pimcore:documents:cleanup printpage printcontainer'</info>");
```
This doesn't require the web2print bundle to be active anymore, since it was suggested that the command would be useful to other types as well and that should reside in the core. 

Adding some quote to the actual command, otherwise could be hard to read wheter the command ends. Not sure if there's any tag like <info> that could highlight it. Maybe some lines break would also work.

Reply:
The problem with this line is that it will never be shown somehow. I left it there because we also do it in the E-Commerce Framework. 

https://github.com/symfony/console/blob/6.2/Formatter/OutputFormatter.php#L73 The output formatter has some the info style but we could also change it to yellow e.g.